tracer/src/Datadog.Trace.Tools.Runner/Checks/ProcessBasicCheck.cs

```diff
@@ -147,7 +165,9 @@ internal static bool CheckRegistry(IRegistryService? registry = null)
                     }
                 }
 
-                return !foundKey;
+                ok &= !foundKey;
```

Comment:
We should also check `SOFTWARE\WOW6432Node` in the code above.

```suggestion
                if (RuntimeInformation.OSArchitecture == Architecture.X64)
                {
                    foreach (var name in registry.GetLocalMachineValueNames(@"SOFTWARE\WOW6432Node\Microsoft\.NETFramework"))
                    {
                        if (suspiciousNames.Contains(name))
                        {
                            Utils.WriteWarning(SuspiciousRegistryKey(name));
                            foundKey = true;
                        }
                    }
                }

                ok &= !foundKey;
```

Reply:
The code above was not changed in this PR, so feel free to tackle this later. AIT-789